internal/kafka/internal/handlers/kafka.go

```diff
@@ -65,6 +67,11 @@ func (h kafkaHandler) Create(w http.ResponseWriter, r *http.Request) {
 
 			convKafka.CloudProvider, convKafka.Region, _ = getCloudProviderAndRegion(ctx, h.service, &kafkaRequestPayload, h.providerConfig)
 
+			// enterprise kafkas should be assigned to specified cluster, if its ID is provided
+			if !shared.StringEmpty(kafkaRequestPayload.ClusterId) {
```

Comment:
Should we use 
```suggestion
			if !shared.StringEmpty(kafkaRequestPayload.ClusterId, true) {
```
so that even `all spaces` is considered `empty` ?

Reply:
No need for this. The `ValidateNotEmptyClusterId` validator already takes care of this.